src/providers/AuthServerProvider.ts

```diff
@@ -127,6 +127,27 @@ export class AuthServerProvider {
     sso.localStorageStoreIdentity(signer, identity)
   }
 
+  /**
+   * Get the persisted account from local storage.
+   */
+  static getAccount = () => {
```

Comment:
Used by the explorer website to determine if the user is already connected and render the flow accordingly